source/index.txt

```diff
@@ -24,6 +24,15 @@ exploring your data in a visual environment, you can also use
 :ref:`manage indexes <compass-indexes>`, and implement
 :ref:`document validation <validation>`.
 
+.. admonition:: MongoDB Atlas
+   :class: note
+
+   `MongoDB Atlas <https://www.mongodb.com/cloud/atlas?tck=docs_compass>`__ 
```

Comment:
What about taking a slightly different approach and highlighting the free sample data? That might help people who want to get started seeing what Compass can do.

We could title the admonition like "Need Data?"

Atlas, MongoDB's cloud-hosted database-as-a-service provides free sample datasets for Atlas clusters. You can use Compass to explore and interact with these sample datasets.

(include link to sample data where appropriate)

Reply:
I like that - wanting to use Compass with sample data is probably pretty common.